Geometry/GlobalTrackingGeometryBuilder/test/BuildFile.xml

```diff
@@ -5,6 +5,7 @@
 <use   name="Geometry/RPCGeometry"/>
 <use   name="Geometry/DTGeometry"/>
 <use   name="Geometry/CSCGeometry"/>
+<use   name="Geometry/MTDGeometryBuilder"/>
```

Comment:
@smuzaffar - IMHO, this is wrong. There should not be any dependencies on the geometry builders.

Reply:
I agree with Yana.

Reply:
@ianna @Dr15Jones so what is the preferred solution here? Integrate the code as it is and then modify the Tracker/MTD structure in next future, or keep this on hold until the geometries have been modified?

Reply:
I'd suggest to restructure before integrating the PR

Reply:
@ianna , this dependency is in tests only. Looks like this unit test is testing many other geometries too.

[a]
```
  void analyzeCSC(const GlobalTrackingGeometry* geo, const CSCGeometry* cscGeometry);
  void analyzeDT(const GlobalTrackingGeometry* geo, const DTGeometry* dtGeometry);
  void analyzeRPC(const GlobalTrackingGeometry* geo, const RPCGeometry* rpcGeometry);
  void analyzeGEM(const GlobalTrackingGeometry* geo, const GEMGeometry* gemGeometry);
  void analyzeMTD(const GlobalTrackingGeometry* geo, const MTDGeometry* mtdGeometry);
  void analyzeTracker(const GlobalTrackingGeometry* geo, const TrackerGeometry* tkGeometry);
```